### ISSUE TITLE
The way people are gaining genescore and becoming leader 

### ISSUE THREAD

**Issue by DiscardedSlinky** [jasonrohrer/OneLife#635]
So the current meta of gaining genescore is:
1.If you're a female leave town immediately and go help another family. It will keep you from losing points over kids.
2.Go AFK in a 1x1 pinewall hut outside of the home radius. Easy to gain score while doing something else.

These methods are not the way a person should be gaining leadership. It's not fair that they gain points for not even being near their family while I spend my entire life at home helping the town and having kids and my score tanks. I've seen people become leader and they're nowhere near home leaving us without a leader truthfully.

I think that people that stay and are actually helpful should be getting more points. I think my entire life of baking and providing for my family should reflect positively on my score. An easy way to fix this is boosting points for having children. If they're worth more people will stay in the homezone for them. Another solution is that people inside the homezone the majority of their life should get more points. People outside the zone should still get points but much less. 

Also leadership should be given to the person CLOSEST with the highest genescore. If a leader is out of town for too long their title should be taken away.

**Comment by Arcurus** [comment]
my idea would be to change gene score the following:
- yourself get only negative 
- your grand kids and grand grand kids give always positive
- the rest is normal

with that you would need to have kids / relatives to get score

solves also that you get punished for your grand grand kids dying early with out you being able to do anything since you are long term dead.


To backing: would be nice if the one who made the food get 20% of the food value, since making food is also fulfilling.

**Comment by Spodnie** [comment]
I keep being around 50 gene score without all these weird practics, so I wouldn't say these are the "meta". I've seen in forums that people are often leaders even tho they don't want it, they don't "farm" their genes either.

I don't see any issue here.

**Comment by jasonrohrer** [comment]
First of all, if you leave town to avoid having children, you will still get impacted by the lives of your own mother, grandmother, and younger sibs that you leave behind.  But perhaps a childless woman should get impacted by nieces and nephews, just like a man would.... I'll look into that.

But yeah, there is obviously a form of "birth control" in the game now, and that's an unintentional side-effect.

Next, if having kids is too risky, that has been fixed this week, with each mother only having a max of 4 living children at one time.  So you can watch them all and take care of them better.

Finally, what's the 1x1 pine hut trick?  Are you saying you can live forever there for some reason?  Or just put on clothes, wait over there, and listen for the DING to feed yourself periodically?

If that's possible, that will also be fixed if a childless woman is impacted by nieces and nephews.

**Comment by MrXIX-OHOL** [comment]
I'm fairly sure that is a way to get perfect temperature without the need of fire.

**Comment by jasonrohrer** [comment]
So... 1x1 pine hut, with pine door and floor?

And fulls set of clothing?

**Comment by jasonrohrer** [comment]
Yes, I see that with full set of clothing, standing in a pine hut gives me about 32 seconds per pip, which is roughly 10 minutes of game time.

That is pretty much working as intended, I think.

Avoiding having kids, to avoid impacts to your gene score, should not be possible, and I'll fix that for sure.

**Comment by TwistedHBG** [comment]
Perhaps if you live a life as a female and you gave birth to no kids you get -0.5 gene score? "Failed to propagate genes" or something? 0.5 might seem kinda harsh but keep in mind that you're already getting points from you living to 60 and your mother living to 60.

I have 49 score and my most recent life I was a man, and I got affected by only two people - I gained 0.116 points from my mother dying at age 50, and I gained 0.22 from myself dying at 60. If I were a woman and I also got punished with -0.5 for having zero kids my total change from that life would have been -0.164, which is a tiny number but it's significant enough to make avoiding having babies not a viable way to get to number one spot.
